### ISSUE TITLE
[feature] Document currently supported and/or eol'ed versions

### ISSUE THREAD

**Issue by funkyfuture** [eXist-db/exist#5410]
when working in the capacity of a system administrator i'm interested whether a (developer) requested deployment will receive security fixes (which i regard as the minimum for what i name "maintained" in the following). at the time of writing it's unclear to me whether that'd be the case for 6.0.1. right now it *seems* to me that three major release lines are supposed to be usable, but with the minor version part the policy/practice seems varying.

i looked around and only found a clue in `SECURITY.md` which seems outdated. it would be helpful to have one of these references available in a place like the Downloads webpage or the README:

- a list of currently maintained minor release lines
- a list of unmaintained minor release lines
- an elaboration of the criteria which minor release lines can be considered as maintained

**Comment by line-o** [comment]
I agree we should provide a list of maintained versions prominently on the homepage.
As of now 6.4.0 is the current maintained release of exist-db with 7.0.0-SNAPSHOT being the scheduled next version.
We do not maintain prior major versions.
